docs/account.mdx

```diff
@@ -0,0 +1,630 @@
+
+# `account`
+
+`account` command is used for consensus or ParaTime-layer on-chain actions
+associated with accounts:
+
+- getting the account balance including delegations,
+- transferring tokens between the accounts,
+- delegating or undelegating tokens to or from validators (*staking*),
+- depositing and withdrawing tokens to or from a ParaTime,
+- changing permission policy of the account,
+- setting commission schedule of the validator account.
+
+## Show
+
+Use `account show [address]` command to print the balance of the account on the consensus layer
+and the currently selected ParaTime, if it is non-zero. Running it without
+arguments will show you the balance of your default account.
+
+```
+$ oasis account show
+Address: oasis1qrec770vrek0a9a5lcrv0zvt22504k68svq7kzve
+Nonce: 28
+
+=== CONSENSUS LAYER (testnet) ===
+  Total: 984.669098086 TEST
+  Available: 984.669098086 TEST
+
+
+  Allowances for this Account:
+    Total: 0.0000002 TEST
+    Allowances:
+      - Beneficiary: oasis1qrydpazemvuwtnp3efm7vmfvg3tde044qg6cxwzx
+        Amount:      0.0000002 TEST
+
+
+=== emerald PARATIME ===
+Balances for all denominations:
+  1.0 TEST
+```
+
+To check the current balance of an arbitrary account, pass the native or
+Ethereum-compatible address.
+
+```
+$ oasis account show oasis1qzzd6khm3acqskpxlk9vd5044cmmcce78y5l6000
+Address: oasis1qzzd6khm3acqskpxlk9vd5044cmmcce78y5l6000
+Nonce: 0
+
+=== CONSENSUS LAYER (testnet) ===
+  Total: 1.0 TEST
+  Available: 1.0 TEST
+
+$ oasis account show 0xBd16C6bF701a01DF1B5C11B14860b6bDbE776669
+Address: oasis1qrvzxld9rz83wv92lvnkpmr30c77kj2tvg0pednz
+Nonce: 0
+
+=== CONSENSUS LAYER (testnet) ===
+  Total: 0.0 TEST
+  Available: 0.0 TEST
+```
+
+To show active staking delegations, pass the `--show-delegations` flag.
+
+```
+$ oasis account show --show-delegations
+Address: oasis1qrec770vrek0a9a5lcrv0zvt22504k68svq7kzve
+Nonce: 28
+
+=== CONSENSUS LAYER (testnet) ===
+  Total: 1006.262158403 TEST
+  Available: 984.669098086 TEST
+
+  Active Delegations from this Account:
+    Total: 21.593060317 TEST
+
+    Delegations:
+      - To:     oasis1qz2tg4hsatlxfaf8yut9gxgv8990ujaz4sldgmzx
+        Amount: 21.593060317 TEST (20000000000 shares)
+
+  Allowances for this Account:
+    Total: 0.0000002 TEST
+    Allowances:
+      - Beneficiary: oasis1qrydpazemvuwtnp3efm7vmfvg3tde044qg6cxwzx
+        Amount:      0.0000002 TEST
+
+
+=== emerald PARATIME ===
+Balances for all denominations:
+  1.0 TEST
+```
+
+:::info
+
+`--show-delegations` flag is not enabled by default, because account
+delegations are not indexed on-chain. To retrieve this information Oasis CLI
+relies on external block explorers such as [oasisscan.com], which may expose
+your privacy.
+
+:::
+
+[oasisscan.com]: https://oasisscan.com
+
+## Network, ParaTime and Account selector
+
+Every `account` subcommand accepts the `--network <network_name>` parameter
+which enable you to specify other than the default [network] the Oasis CLI
```

Comment:
Mentions the notion of a default network, should it link to docs showing how to change defaults (maybe in a tip box)?